bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/airqHandler.java

```diff
@@ -0,0 +1,936 @@
+                    changeSettings(newobj);
+                    break;
+                case "nightmode_WifiNightOff":
+                    subjson.addProperty("WifiNightOff", ohCmd2airqCmd(command.toString()));
+                    newobj.add("NightMode", subjson);
+                    changeSettings(newobj);
+                    break;
+                case "SSID":
+                    JsonElement wifidatael = new Gson().fromJson(command.toString(), JsonElement.class);
+                    if (wifidatael != null) {
+                        JsonObject wifidataobj = wifidatael.getAsJsonObject();
+                        newobj.addProperty("WiFissid", wifidataobj.get("WiFissid").getAsString());
+                        newobj.addProperty("WiFipass", wifidataobj.get("WiFipass").getAsString());
+                        String bssid = wifidataobj.get("WiFibssid").getAsString();
+                        if (!bssid.isEmpty()) {
+                            newobj.addProperty("WiFibssid", bssid);
+                        }
+                        newobj.addProperty("reset", wifidataobj.get("reset").getAsString());
+                        logger.info("air-Q - airqHandler - handleCommand(): dummy to change settings: {}",
+                                newobj.toString());
+                        changeSettings(newobj);
+                    } else {
+                        logger.error("Cannot extract wlan data from this string: {}", wifidatael);
+                    }
+                    break;
+                case "timeServer":
+                    newobj.addProperty(channelUID.getId(), command.toString());
+                    changeSettings(newobj);
+                    break;
+                case "nightmode_StartDay":
+                    if (isTimeFormat(command.toString())) {
+                        subjson.addProperty("StartDay", command.toString());
+                        newobj.add("NightMode", subjson);
+                        changeSettings(newobj);
+                    } else {
+                        logger.error(
+                                "air-Q - airqHandler - handleCommand(): {} should be set to {} but it isn't a correct time format (eg. 08:00)",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                case "nightmode_StartNight":
+                    if (isTimeFormat(command.toString())) {
+                        subjson.addProperty("StartNight", command.toString());
+                        newobj.add("NightMode", subjson);
+                        changeSettings(newobj);
+                    } else {
+                        logger.error(
+                                "air-Q - airqHandler - handleCommand(): {} should be set to {} but it isn't a correct time format (eg. 08:00)",
+                                channelUID.getId(), command.toString());
+                    }
+                    break;
+                // TODO
```

Comment:
Is there is missing code here? remove comment if this is old.

Reply:
Included now. I had the code in another, OH2 version, and forgotten to port it.